app/views/idv/gpo_verify/index.html.erb

```diff
@@ -11,7 +11,10 @@
 
 <% title t('titles.verify_profile') %>
```

Comment:
Do we want to update the browser tab title to match the new H1 heading text?

Reply:
good question! @Kamal-Munshi ?

Reply:
@aduth we confirmed with Princess who handles content that leaving it as 'Verify Profile' title is more descriptive than 'Welcome back'.

Reply:
OK, but it's not 'Verify Profile', it's 'Activate your account'. Is that still what we want?

Reply:
She meant the current title says Verify Profile and to keep it. Want me to ask about Activate your account?

Reply:
Might as well ask, but to be clear, the current title does not say "Verify Profile".

Reply:
Changed to Activate your account.